libsolidity/interface/CompilerStack.h

```diff
@@ -498,6 +502,7 @@ class CompilerStack: public langutil::CharStreamProvider
 	std::map<std::string const, Source> m_sources;
 	// if imported, store AST-JSONS for each filename
 	std::map<std::string, Json::Value> m_sourceJsons;
+	std::map<std::string, Json::Value> m_evmAssemblyJson;
```

Comment:
Please document. Also I'm not super happy to have this here...

Reply:
Yeah, I know.. I was also not very sure whether I should just do it like this. Will think about this.

Reply:
Ok, I just removed that. Current version will use only `m_sourceJsons`.